scipy/interpolate/tests/test_interpolate.py

```diff
@@ -1523,7 +1527,7 @@ def test_roots_idzero(self):
         x = np.array([0, 0.4, 0.6, 1.0])
 
         pp = PPoly(c, x)
-        assert_array_equal(pp.roots(),
+        xp_assert_equal(pp.roots(),
                            [0.25, 0.4, np.nan, 0.6 + 0.25])
```

Comment:
I haven't commented on other places like this to avoid nitpicking (based on prior feedback), but it would be nice to be careful with indentation like this in future PRs.

Reply:
thanks, fixed.